6.4.rememberRemember.py

```diff
@@ -0,0 +1,15 @@
+from PIL import Image

```

Comment:
This script doesn't work as expected.
Fix it and resubmit in order to improve your grade.